setup.py

```diff
@@ -22,10 +22,9 @@
     test_suite="nose.collector",
     classifiers=[
         "License :: OSI Approved :: MIT License",
-        "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.7",
```

Comment:
this means it only works only on 3.7
you should list it works with the different python 3 versions we test on the CI

Reply:
yes